### ISSUE TITLE
Add the lang option

### ISSUE THREAD

**Issue by pepelsbey** [shower/cli#50]
Something like this:

```html
<!DOCTYPE html>
<html lang="<%= lang %>">
```

It’s important to have a proper language set.

**Comment by nikolay-govorov** [comment]
Yes, that sounds great. What do you think this menu item should look like?

**Comment by pepelsbey** [comment]
Something like

```
? Select language
❯ English
  Russian
  Custom…
```

That would give you `lang="en"` or `lang="ru`. In “Custom…” case you’ll be prompted to enter either language name or language code, depending on how you’re planning to implement it.

**Comment by nikolay-govorov** [comment]
There are a few details that I would like to consult about.

1. Should the default language be English, or maybe the current system language?
2. Whether to validate the language that the user enters in the custom? Probably by [bcp47](https://tools.ietf.org/html/bcp47)

**Comment by pepelsbey** [comment]
It depends on how much time you want to spend working on this feature :) You can always just go for En, Ru, Custom… and keep developers responsible for their choice.

But as the next enhancement I’d validate the Custom value and only then work on guessing the default option.
